addons/web_editor/static/src/xml/editor.xml

```diff
@@ -114,7 +114,9 @@
             </div>
 
             <div id="link" class="btn-group">
+                <div id="media-modal" title="Insert or edit media" class="fa fa-file-image-o fa-fw btn editor-ignore"></div>
```

Comment:
Are you sure the media buttons should be in the same button group as the link button? If so then we should change the id of the button group. If you do that, make sure to change references to it in template extensions.

Reply:
I just checked the mockup provided from FP in the channel. 
I'm not sure where this should be located.

Anyways, it would not be a big deal to reorganize it later.